src/App.js

```diff
@@ -2,26 +2,45 @@ import React, { Component } from 'react';
 import './App.css';
 import movieData from './mock-data';
 import Nav from './components/Nav/Nav.js';
-import Hero from './components/Hero/Hero.js';
 import Movies from './components/Movies/Movies.js';
-import Carousel from './components/Carousel/Carousel.js'
-
+import Carousel from './components/Carousel/Carousel.js';
+import fakeMovieDetailData from './fake-movie-detail-data.js';
+import MovieDetail from './components/MovieDetail/MovieDetail.js';
 class App extends Component {
   constructor() {
     super();
     this.state = {
       movies: movieData.movies,
+      // movies: [],
+      selectedMovieDetails: {},
     }
   } 
 
+  displayMovieDetail = (id) => {
+    // fetch(id) 
+    // ... returns detail data
+    this.setState({ selectedMovieDetails: fakeMovieDetailData.movie });
+  }
```

Comment:
good use of set state here.  I was reading over the rubric last night and it mentioned having all the api calls in a separate file. So maybe we would import the function to this method?

Reply:
Okay, so maybe we'll have something like this?
```
const myFetch = fetch(blah blah)
  .then(response => response.json())
```
and then here we'd have
```
myFetch()
.then(data => this.setState({ selectedMovieDetails: data }))
```